src/Form/test-cases/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+    ['poor-markup-form', require('./poor-markup-form')],
+    ['etsy', require('./esty')]
+]
```

Comment:
As we're running in a node context here, can we make the requires file paths instead? They could just be flat HTML files then.

```
module.exports = [
    ['poor-markup-form', './poor-markup-form.html'],
    ['etsy', './etsy.html']
]
```
Potentially you could just have the file name as the test case name so:
```
module.exports = [
    './poor-markup-form.html',
    './etsy.html'
]
```

And if we were to adopt my suggestion this could look like:
```
module.exports = [
    './poor-markup-form.html': {
        "some > selector": "cardNumber",
        "#fieldname": "expiration",
        "#otherFieldName": "cardSecurityCode"
    }
    './etsy.html': {
       ...
    }
]
```